fastcrypto/src/groups/bls12381.rs

```diff
@@ -749,7 +749,7 @@ impl ScalarType for Scalar {
 ///
 /// The input buffer must be at least 48 bytes long to ensure that there is only negligible bias in
 /// the output.
-pub(crate) fn reduce_mod_uniform_buffer(buffer: &[u8]) -> Scalar {
+pub fn reduce_mod_uniform_buffer(buffer: &[u8]) -> Scalar {
     assert!(buffer.len() >= 48);
```

Comment:
If this is to be exposed publicly, I think it should return an error if the input is too small instead of panicking.

Reply:
revised for returning result instead of panic for public function. this is used for bls key derivation SIP impl.